File: PERL/Cap05/c05ex11.pl
# c05ex11.pl

#!/usr/bin/perl

use strict;
use warnings;

for (my $I = 1; $I <= 5; $I++) {
  printf "Entre um valor numerico: ";
  my $N = <STDIN>;
  chomp $N;

  my $R = $N * 3;

  print "Resultado = $R\n";
  print "\n";
}

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: C/Cap05/c05ex11.c
// c05ex11.c

#include <stdio.h>

int main(void) {
  int I, N, R;

  for (I = 1; I <= 5; I++) {
    printf("Entre um valor numerico: ");
    scanf("%d", &N);
    while (getchar() != '\n');
    
    R = N * 3;

    printf("Resultado = %d\n\n", R);
  }

  printf("\n");
  printf("Tecle <Enter> para encerrar... ");
  while (getchar() != '\n');

  return 0;
}
